plugins/scaffolder/src/components/fields/RepoUrlPicker/AzureRepoPicker.tsx

```diff
@@ -55,15 +57,29 @@ export const AzureRepoPicker = (props: {
         error={rawErrors?.length > 0 && !organization}
       >
         {allowedOrganizations?.length ? (
-          <Select
-            native
-            label="Organization"
-            onChange={s =>
-              onChange({ organization: String(Array.isArray(s) ? s[0] : s) })
-            }
-            disabled={allowedOrganizations.length === 1}
-            selected={organization}
-            items={organizationItems}
+          <Autocomplete
+          aria-label="Organization"
+          options={organizationItems}
+          getOptionLabel={(organizations) => organizations.label || "error getting label"}
+          disabled={allowedOrganizations.length === 1}
+          data-testid="select"
+          freeSolo
```

Comment:
Not sure what we want to have `freeSolo` on here if there's `allowedOrgs`

Reply:
Same for all the other autocompletes

Reply:
Then I will need a bit of help with state management @benjdlambert if you do not mind. Am a bit stuck with onChange